MaterialProgressBar/src/main/ets/components/MaterialProgressBar.ets

```diff
@@ -0,0 +1,173 @@
+/*
+ * Copyright (C) 2022 Application Library Engineering Group
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import MaterialComponent from './basic/MaterialComponents'
+import MaterialProgressStyle from './materialprogressstyle'
+
+@Component
+struct MaterialProgressBar {
+  @State mStyle: MaterialProgressStyle = MaterialProgressStyle.MaterialHorizontal
+  modelBuild: any
+
+  aboutToAppear() {
+    this.modelBuild.style = this.mStyle
+  }
+  onPageShow() {}
+
+  build() {
+    if (this.modelBuild) {
+      if (this.mStyle == MaterialProgressStyle.MaterialHorizontal  || this.mStyle == MaterialProgressStyle.MaterialCircular) {
+        MaterialComponent(this.modelBuild)
+      } else {
+        MaterialComponent(this.modelBuild)
```

Comment:
no change in code of if and else? both does 
 MaterialComponent(this.modelBuild)
Please recheck N correct

Reply:
Resolved